about_sigils.exs

```diff
@@ -14,21 +14,20 @@ defmodule AboutSigils do
   end
 
   think "The ~S sigil is similar to ~s but doesn't do interpolation" do
-    assert_?  ~S[1 + 1 = #{1+1}] == ~s[1 + 1 = #{1+1}]
-    assert_?  ~S[1 + 1 = #{1+1}] == "1 + 1 = \#\{1+1\}"
+    assert ~s[1 + 1 = #{1+1}] == __?
+    assert ~S[1 + 1 = #{1+1}] == __?
```

Comment:
I don't know if somebody learning elixir knows about `\` inside a string to escape special characters. Maybe we should drop a hint there?